openstudiocore/src/utilities/core/Path.hpp

```diff
@@ -22,11 +22,12 @@
 
```

Comment:
One note of caution regarding rolling some of these things together. Some of the path.hpp stuff was carefully crafted so that it could be included in GenerateIddFactory and other top level things and not accidentally cause rebuilds of the entire system & have few dependencies. This is one reason I only very lightly touched those headers.